packages/phoenix-event-display/src/tests/lib/models/cut.model.test.ts

```diff
@@ -0,0 +1,23 @@
+import { Cut } from '../../../lib/models/cut.model';
+
+describe('Cut', () => {
+  let model: Cut;
+
+  beforeEach(() => {
+    model = new Cut('test', 0, 1, 1);
+  });
+
+  it('should create an instance of Cut model', () => {
+    expect(model).toBeTruthy();
+  });
+
+  it('should reset the values', () => {
+    model.reset();
```

Comment:
The values will be reset by default. What we can do here is first assign the values and then reset them.

```ts
it('should reset the values', () => {
  model.minValue = 10;
  model.maxValue = 20;

  model.reset();

  expect(model.minValue).toBe(0);
  expect(model.maxValue).toBe(1);
});
```

Reply:
Sure, changing...